src/alfasim_sdk/testing/fixtures.py

```diff
@@ -162,6 +163,10 @@ def run(self, *, number_of_threads: int = -1) -> Results:
                 ],
                 cwd=project_folder,
             )
+        except CalledProcessError:
+            incorrect_results = Results(self.alfacase_data_folder)
+            if incorrect_results is not None:
+                print(incorrect_results.log.read_text())
```

Comment:
Perhaps print the `log_calc` file also.
What do you think, @prusse-martin ?

Reply:
I have thought  that `log_calc` is not created when the simulation fail.

Reply:
It creates the log_calc file also, at least for the cases I have seen. I have added the log_calc.txt content to this printing. But I want your opinion if does it makes sense, because the log_calc.txt output can be large depending on the situation.

Reply:
@ggrbill what do you think about including the log_calc.txt in this exception handling?

Reply:
I think it is not a problem, since this file is short when the simulation fails. In most cases the simulation will fail right in the beginning, so the file could be printed.